model/src/test/test_model_basic_renewal.py

```diff
@@ -214,3 +233,6 @@ def test_model_basicrenewal_padding() -> None:  # numpydoc ignore=GL08
     # For now the assertion is only about the expected number of rows
     # It should be about the MCMC inference.
     assert inf_mean.to_numpy().shape[0] == 500
+
+
+test_model_basicrenewal_plot()
```

Comment:
These calls at the end of pytest files are not the standard pattern. Are they included for interactive debugging? Another reason?

Reply:
Yes, interactive debugging. I thought I got rid of all those! The regex `^test_` didn't catch it, but `^\s*test_` did. Addressed in 39cec88.